src/TriggerDeliveryPoliciesPage.tsx

```diff
@@ -80,6 +80,7 @@ export default (): React.ReactElement => {
               <Button
                 variant="link"
                 className="p-0"
+                disabled={!astarte.token?.can('realmManagement', 'POST', '/policies')}
```

Comment:
It would be better to hide the button instead of displaying it disabled.

In addition, in the list of policies, we could just display the name of each policy instead of a link when the permission is missing:
```tsx
astarte.token?.can('realmManagement', 'GET', `/policies/${name}`)
```